docs(decluttering): improve information (#1047)
Lundalogik/lime-elements: src/decluttering.md

# Reducing UI clutter and cognitive load

## The philosophy | Minimalism
From our point of view, a great product or tool is essentially one that has a minimalistic user interface. This does not limit to digital products, but it becomes extra important as they are often packed with tools, features, and functionalities.

Both given the aesthetics and functionality concerns, a minimalist product should be reduced to its necessary elements. This can be achieved by usage of fewer design elements, less ornamental elements in its visual design, fewer options and features, and less occupied screen space; as well as carefully breaking the interaction flows or functionalities into distinct and meaningful parts.

Thus when designin a product, we have to keep removing elements and questioning their existence in the UI. We should always keep asking questions such as: Is this really needed? Does this really have to be there? Why? What's the trade-off if it's removed? Do we lose functionality? Do we lose usability? And so forth
> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.
>

> > – Antoine de Saint-Exupéry

Reviewer: Haha! As I was reading this, I was thinking of this quote, and that it would fit nicely in this section. Then I got to this 👌 🤣

Author: 😅

Such cases should be accompanied with other clear visual hints where the
errors or progress-blockers are located in the UI.
:::

> See how Lime Elements can automatically hide buttons when they are not needed, in the "Reduce Presence" section of our [Button component](#/component/limel-button/).

Reviewer: ```suggestion
See how Lime Elements can automatically hide buttons when they are not needed, in the "Reduce Presence" example for [limel-button](#/component/limel-button/).
```
